src/main/java/us/ilite/robot/auto/PracticeTrajectory.java

```diff
@@ -0,0 +1,37 @@
+package us.ilite.robot.auto;
+
+import java.util.ArrayList;
+
+import edu.wpi.first.math.trajectory.TrajectoryGenerator;
+
+import edu.wpi.first.math.geometry.Pose2d;
+import edu.wpi.first.math.geometry.Rotation2d;
+import edu.wpi.first.math.geometry.Transform2d;
+import edu.wpi.first.math.geometry.Translation2d;
+import edu.wpi.first.math.trajectory.TrajectoryConfig;
+import edu.wpi.first.math.util.Units;
+
+public class PracticeTrajectory {
+    //This is the 2018 code
+    public void generateTrajectory() {
```

Comment:
Based on our conversation, let's change this method to return a  Trajectory. I'd also recommend creating an interface called ITrajectoryFactory that has a single method that is this generateTrajectory method. This way we can have multiple factories.